test-suite-java/src/test/java/io/micronaut/sourcegen/example/AnimalSuperBuilderTest.java

```diff
@@ -61,6 +64,13 @@ public void testDog() {
     }
 //end::test[]
 
+    @Test
+    public void dogIntrospection() {
+        var introspection = BeanIntrospection.getIntrospection(DogSuperBuilder.class);
+        assertNotNull(introspection);
+        assertEquals(6, introspection.getBeanProperties().size());
```

Comment:
@graemerocher  I could not make introspection work for the `@SuperBuilder` case.

For `@Builder` I just added `@Creator` on the PersonBuilder all-args contructor which creates a correct introspection. Here there is no all-args constructor, so I tried `@Introspected(withPrefix = "")` on `DogBuilder`, but that does not help resolving the properties.

Do you have any ideas?

Note, that I want to configure introspection for builder separately since this does not work:
```
@Introspected(builder = @IntrospectionBuilder(
    builderClass = DogSuperBuilder.class
))
public class Dog {
```
```
Builder class not found on compilation classpath: <error>
```

Reply:
Actually, solved it by adding an all-args constructor for the `SuperBuilder`, too